doc/man7/EVP_KEM-RSA.pod

```diff
@@ -0,0 +1,65 @@
+=pod
+
+=head1 NAME
+
+EVP_KEM-RSA
+- EVP_KEM RSA keytype and algorithm support
+
+=head1 DESCRIPTION
+
+The B<RSA> keytype and its parameters are described in L<EVP_PKEY-RSA(7)>.
+See L<EVP_PKEY_encapsulate(3)> and L<EVP_PKEY_decapsulate(3)> for more info.
+
+=head2 RSA KEM parameters
+
+=over 4
+
+=item "operation" (B<OSSL_KEM_PARAM_OPERATION>) <UTF8 string>
+
+The RSA Key Exchange Mechanism only currently supports the following operation
```

Comment:
Key Encapsulation Mechanism?

Reply:
Strictly speaking an external provider could support other KEMs so this is true only for our own providers.

Reply:
That is incredibly picky :) This is about the Openssl implementation.
You could say the same thing about any pod file.. Changed the wording.